### DISCUSSION TITLE
圖靈測試

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1115] [General]
對不起可能是我孤陋寡聞
今天才真正去查圖靈測試是什麼
查完之後想的第一件事情是
說不定某些主播或小網紅可以買高度擬人的聊天機器人
（或其實現在真的有人會這麼做
這樣不僅可以把時間分給其他事還同時可以賺錢
（挖勒會不會太壞哈哈哈
https://wiki.mbalib.com/zh-tw/图灵测试

**Comment by thomasxshen** [comment]
圖靈測試，由英國電腦科學家艾倫·圖靈於1950年提出，是一種評估機器是否能展現與人類不可區分的智能的思想實驗。在這個測試中，一位詢問者會向隔離在另一個房間的一個人和一台機器提問，並根據回答來判斷哪一個是人類，哪一個是機器。所有的交流都是通過文字進行，排除了語音辨識的可能性。

**Comment by jeff110303549** [comment]
圖靈測試雖然是一個最有名的測試，卻也是一個有爭議的測試方法，尤其是準確性存在著爭議。一些人認為，圖靈測試是一個有用的指標，可以衡量機器是否能夠表現出人類智能。然而，其他人則認為圖靈測試存在一些缺陷，無法完全準確地評估機器的智能。一些圖靈測試的批評點包括：標準主觀性、過於狹義、容易受騙、技術限制等
圖靈測試並不能完全準確地反映機器的智能程度。因此，在評估機器智能時，還需要綜合考慮其他因素，如機器在特定任務上的表現和使用者的實際體驗等，才能夠有更準確的表現

**Comment by YuanWang08** [comment]
其實國外有一個新聞，內容大概是一個工程師用AI模仿自己的說話模式，同時使用交友軟體與多位女性交談，然後他在從中選擇自己喜歡的真正出去會面，就它自身所述，會面後它告知對方其實之前都是AI在代替他聊天，大部分的人都表示感覺不出來，聽到的時候大吃一驚，先不論道德上的問題，至少這世界上已經有人無法分辨出AI與人類了。
